external/xamarin-android-tools.override.props

```diff
@@ -0,0 +1,4 @@
+<Project>
+  <!-- Import $(LibZipSharpVersion) -->
+  <Import Project="..\Directory.Build.props" />
```

Comment:
Should I move this section to a new file?

https://github.com/xamarin/xamarin-android/blob/73684872b3891dbaf7cc27d7675ac5f1b813160c/Directory.Build.props#L42-L56

(Then we'd import less)

Reply:
We don't think any of the properties in here hurt anything.